xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -763,7 +763,7 @@ std::string CGUIWindowVideoBase::GetResumeString(const CFileItem &item)
 
 bool CGUIWindowVideoBase::ShowResumeMenu(CFileItem &item)
 {
-  if (!item.m_bIsFolder && !item.IsPVR())
+  if (!item.m_bIsFolder && !item.IsPVR() || item.IsPlugin())
```

Comment:
yes, just confirmed, item.m_bIsFolder is True when an addon is called from the Info screen.